src/qt/walletmodel.cpp

```diff
@@ -65,6 +65,15 @@ WalletModel::~WalletModel()
     unsubscribeFromCoreSignals();
 }
 
+QString WalletModel::getWalletName() const
```

Comment:
In commit "Qt: When multiple wallets are used, include in notifications the name"

Would be nice to follow this up by removing the `name` parameter passed to `BitcoinGUI::addWallet` to avoid duplicating .dat stripping logic.

Reply:
Done.